src/Microsoft.TemplateEngine.Edge/Settings/Scanner.cs

```diff
@@ -54,14 +55,13 @@ private MountPointScanSource GetOrCreateMountPointScanInfoForInstallSource(strin
                     bool isLocalFlatFileSource = mountPoint is FileSystemMountPoint
                                                 && !sourceLocation.StartsWith(_paths.ScratchDir);
 
-                    return new MountPointScanSource()
-                    {
-                        Location = sourceLocation,
-                        MountPoint = mountPoint,
-                        ShouldStayInOriginalLocation = isLocalFlatFileSource,
-                        FoundTemplates = false,
-                        FoundComponents = false,
-                    };
+                    return new MountPointScanSource(
```

Comment:
I found such constructors difficult to read, it would be good to add parameter names, especially to "true"